lib/wor/paginate/formatter.rb

```diff
@@ -0,0 +1,38 @@
+module Wor
+  module Paginate
+    class Formatter
+      attr_accessor :adapter, :content, :formatter, :options
+
+      def initialize(adapter, options = {})
+        self.adapter = adapter
+        self.options = options
+      end
+
+      def format
+        { page: serialized_content, count: count, total_pages: total_pages,
```

Comment:
Why not ?

    {
      page: serialized_content,
      count: count,
      total_pages: total_pages,
      total_count: total_count,
      current_page: current_page,
      next_page: next_page
    }